path_supplier/graphhopper_path_supplier.go

```diff
@@ -10,9 +10,9 @@ import (
 )
 
 // precision to around .0001 deg, which corresponds to max 10m
-const URL_FORMAT = "%s%s?point=%.6f,%.6f&point=%.6f,%.6f&vehicle=%s&key=%s&points_encoded=false"
-const GRAPHHOPPER_API_URL_BASE = "https://graphhopper.com/api/1"
-const ROUTING_API_URL = "/route"
+const urlFormat = "%s%s?point=%.6f,%.6f&point=%.6f,%.6f&vehicle=%s&key=%s&points_encoded=false"
```

Comment:
maybe let's follow convention to have snake_case private constants and screaming snake_case public constants